bank/persistence/user_operations_test.go

```diff
@@ -31,7 +31,7 @@ func TestItStartsUpdatesAndDeletesSessionFromDB(t *testing.T) {
 	sessionStore.UpdateSession(userSession.SessionID, expected.Expires)
 
 	var result domain.Session
-	session.DB("bankTest").C("sessions").Find(bson.M{"userid": testUser.UserID}).One(&result)
+	db.Database.C("sessions").Find(bson.M{"userid": testUser.UserID}).One(&result)
```

Comment:
Smelly usage of mgo's api in tests. 

findAvailableAt function should return session if it exists  
findAvailableAt function should not return session when it doesn't exists